lib/ansible/plugins/action/__init__.py

```diff
@@ -982,7 +993,7 @@ def _low_level_execute_command(self, cmd, sudoable=True, in_data=None, executabl
         # be sure to remove the BECOME-SUCCESS message now
         out = self._strip_success_message(out)
 
-        display.debug(u"_low_level_execute_command() done: rc=%d, stdout=%s, stderr=%s" % (rc, out, err))
+        display.debug(u"_low_level_Execute_command() done: rc=%d, stdout=%s, stderr=%s" % (rc, out, err))
```

Comment:
This line looks like it only introduces a typo ;-)

Reply:
last minute updates ...